tailwind.config.js

```diff
@@ -2,7 +2,39 @@ module.exports = {
   purge: ['./src/**/*.{js,jsx,ts,tsx}'],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {}
+    fontSize: {
```

Comment:
Configured the theme by following [this guideline](https://tailwindcss.com/docs/theme)

Reply:
@rafayet-monon It's the values for `font-size` and default `line-height` according to [this](https://tailwindcss.com/docs/font-size#providing-a-default-line-height) 🙌🏼